src/OSPSuite.SimModelNative/src/Species.cpp

```diff
@@ -63,7 +63,7 @@ void Species::SetODEScaleFactor (double p_ODEScaleFactor)
 void Species::LoadFromXMLNode (const XMLNode & pNode)
 {
 	// ---- XML sample
-	//<Species Id="S1" Name="Drug" Path="Liver/Cells" InitialValueFormulaId="F2" Unit="�mol/l">
+	//<Species Id="S1" Name="Drug" Path="Liver/Cells" InitialValueFormulaId="F2" Unit="�mol/l">
```

Comment:
What's this?

Reply:
In fact all files should be stored as UTF-8 anyways, especially if they have special characters like this one

Reply:
This is the only way how unicode works for me: 

![image](https://github.com/user-attachments/assets/687ff04a-b06e-492a-81ab-67133ccebb69)

Turning the experimental support ALWAYS messes up UTF-8 support for me :D

Reply:
Without this option activated I had other issues (crashes!) with the ospsuite-R package (s. the discussion in https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/1480) - which I could fix only by activating the option (following the advice by Felix, https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/1480#issuecomment-2306477359)